drizzlepac/hapsequencer.py

```diff
@@ -92,6 +100,9 @@ def create_catalog_products(total_obj_list, log_level, diagnostic_mode=False, ph
         Which algorithm should be used to generate the sourcelists? 'aperture' for aperture photometry;
         'segment' for segment map photometry; 'both' for both 'segment' and 'aperture'. Default value is 'both'.
 
+    catalog_switches : dict, optional
+        Specify which, if any, catalogs should be generated at all, based on detector.
+
```

Comment:
Please clarify the possible value of the switches and their corresponding meaning.

Reply:
See changes in 65312f8.